pybleau/app/model/tests/test_plot_template_manager.py

```diff
@@ -80,3 +79,18 @@ def test_rescan_template_dir_raises_event(self):
     def test_get_names_returns_list(self):
         names = ["temp1", "temp2"]
         self.assertCountEqual(names, self.manager.names)
+        os.remove(self.template_paths[0])
+        self.manager.rescan_template_dir()
+        names = ["temp2"]
+        self.assertCountEqual(names, self.manager.names)
+        os.remove(self.template_paths[1])
+        self.manager.rescan_template_dir()
+        names = []
+        self.assertCountEqual(names, self.manager.names)
```

Comment:
Suggestion: compactify line 88 and 89 together: the `names` variable isn't useful. Same suggestion for lines 84 and 85.

Reply:
Will do.